packages/kmod-6.1-nvidia-r570/kmod-6.1-nvidia-r570.spec

```diff
@@ -47,13 +47,17 @@ Source206: nvidia-persistenced.service
 # NVIDIA tesla conf files from 300 to 399
 Source300: nvidia-tesla-tmpfiles.conf
 Source301: nvidia-tesla-build-config.toml.in
-Source302: nvidia-open-gpu-config.toml.in
-Source303: nvidia-open-gpu-copy-only-config.toml.in
-Source304: nvidia-ld.so.conf.in
-Source305: link-tesla-kernel-modules.service.in
-Source306: load-tesla-kernel-modules.service.in
-Source307: copy-open-gpu-kernel-modules.service.in
-Source308: load-open-gpu-kernel-modules.service.in
+Source302: nvidia-ld.so.conf.in
+Source303: nvidia-open-gpu-config.toml.in
+Source304: nvidia-open-gpu-copy-only-config.toml.in
+Source305: nvidia-grid-config.toml.in
+Source306: nvidia-grid-copy-only-config.toml.in
+Source307: link-tesla-kernel-modules.service.in
+Source308: load-tesla-kernel-modules.service.in
+Source309: copy-open-gpu-kernel-modules.service.in
+Source310: load-open-gpu-kernel-modules.service.in
+Source311: copy-grid-kernel-modules.service.in
+Source312: load-grid-kernel-modules.service.in
```

Comment:
Source renumbering is always painful because of the use of `%{S:...}` below.

I would say either don't do it, or renumber them according to some scheme, like `driverdog` configs start with `4xx` and service units start with `5xx` - so that the reviewer can confirm visually that the files are still going to the right place.

Reply:
I had them reordered and then a rebase unwound it, I'll fix it. I might move them out to be more separated like you recommend.

Reply:
I did this, I like the outcome! Thanks for the recommendation!